pytest_spec/patch.py

```diff
@@ -54,6 +59,18 @@ def _print_class_information(self):
     self._first_triggered = True
 
 
+def _remove_class_prefix(nodeid):
+    return re.sub("^Test", "", nodeid)
+
+
+def _split_words(nodeid):
```

Comment:
Yeah, I went back and forth on that one. Went with "the name should say *what* the method does, not *how*", but don't have a strong opinion on it.